### ISSUE TITLE
`bridges()` returns memory addresses instead of objects for refvertexed graphs

### ISSUE THREAD

**Issue by merkys** [graphviz-perl/Graph#29]
`bridges()` returns memory addresses instead of objects for refvertexed graphs. Minimal failing example:
```perl
use strict;
use warnings;
use Data::Dumper;
use Graph::Undirected;

my $g = Graph::Undirected->new( refvertexed => 1 );
$g->add_edge( { name => 'A' }, { name => 'B' } );
print $Graph::VERSION, "\n";
print Dumper [ $g->edges ];
print Dumper [ $g->bridges ];
```
Output:
```
0.9726
$VAR1 = [
          [
            {
              'name' => 'A'
            },
            {
              'name' => 'B'
            }
          ]
        ];
$VAR1 = [
          [
            'HASH(0x5575f773e250)',
            'HASH(0x5575f7b67f48)'
          ]
        ];
```
I would expect `$g->bridges` to return the same as `$g->edges` (without respect to order).

**Comment by mohawk2** [comment]
Thanks for the report! I can confirm the issue, and will work soon on a solution.

**Comment by mohawk2** [comment]
What was going on here wasn't quite "returning memory addresses", but inappropriately stringifying the vertex values due to that part of the code naively treating vertices as strings (in hash keys). Now the underlying vertex IDs (in the AdjacencyMap) are used for that, and only converted back when used as vertices.

**Comment by mohawk2** [comment]
The above was released on 25 Jun as 0.9727.
